src/Symfony/Component/RemoteEvent/Attribute/AsRemoteEventConsumer.php

```diff
@@ -18,6 +18,9 @@
 class AsRemoteEventConsumer
 {
     public function __construct(
+        /**
+         * The name of the remote event consumer, used to identify it when defining remote events.
```

Comment:
all these inline comments should be turned into `@param` annotations

Reply:
Forgot about this PR, will update accordingly with what's been done lately. Thanks!